core/js/tests/specs/coreSpec.js

```diff
@@ -560,7 +579,26 @@ describe('Core base tests', function() {
 		});
 	});
 	describe('Util', function() {
+		var locale;
+		var localeStub;
+
+		beforeEach(function() {
+			locale = OC.getLocale();
```

Comment:
Use `OC.getCanonicalLocale()` here.

```suggestion
			locale = OC.getCanonicalLocale();
```